packages/api-explorer/src/utils/path.ts

```diff
@@ -128,3 +132,35 @@ export const getSpecKey = (location: HLocation | Location): string | null => {
   }
   return match?.groups?.specKey || null
 }
+
+/**
+ * Gets the scene type of the current page
+ * @param location browser location
+ * @returns string representing the scene type
+ */
+export const getSceneType = (location: HLocation | Location) => {
```

Comment:
I'd personally prefer if you passed in path as a string in here and performed the regex match on it.